.github/actions/fetch_ctk/action.yml

```diff
@@ -9,15 +9,22 @@ inputs:
   cuda-version:
     required: true
     type: string
+  cuda-components:
+    description: "A list of the CTK components to install as a comma-separated list. e.g. 'cuda_nvcc,cuda_nvrtc,cuda_cudart'"
+    required: false
+    type: string
+    default: "cuda_nvcc,cuda_cudart,cuda_nvrtc,cuda_profiler_api,cuda_cccl,cuda_sanitizer_api,libnvjitlink"
```

Comment:
I have added `compute-sanitizer` to the defaults. It is part of the `cuda_sanitizer_api` component.

Reply:
If it is not too big let's do it. Note that GitHub Cache space is pretty limited (10GB) and there is no paid option to increase it.

Reply:
The cuda_santizier_api component is 9.5MB, so that's probably fine.